### PR TITLE
Update storage and serializable config

### PR DESCRIPTION
**Pull request by yito88**
Updated storage and serializable configurations since Scalar DB configurations have been updated.
I expect that test logic isn't modified at all.

### CODE REVIEW

**Review by feeblefakie — APPROVED** [review]
LGTM 👍

File: scalardl-test/verification-config.toml [hunk 1]
```diff
@@ -24,7 +24,13 @@
   is_verification = true
   num_accounts = 10
   population_concurrency = 32
+
+[storage_config]
   contact_points = "localhost"
```
Comment (brfrn169) [inline scalardl-test/verification-config.toml:29]: It looks like we need to change the verification script side for this change?

**Review by brfrn169 — COMMENTED** [review]
LGTM! Just left one comment. Please take a look at it when you get a chance.

File: scalardl-test/verification-config.toml (hunk 1 above)
Comment (yito88) [reply scalardl-test/verification-config.toml:29]: @brfrn169 Thank you for the suggestion!
As you said, yes, it's needed. I made a PR https://github.com/scalar-labs/scalar/pull/551 .

Comment (brfrn169) [reply scalardl-test/verification-config.toml:29]: Thank you!

**Review by brfrn169 — APPROVED** [review]
LGTM!

**Pull request merged**